internal/cloudapi/v2/openapi.v2.yml

```diff
@@ -778,6 +778,11 @@ components:
         - vsphere
     Repository:
       type: object
+      description: |
+        Repository configuration.
+        At least one of the 'baseurl', 'mirrorlist', 'metalink' properties must
+        be specified. If more of them are specified, the order of precedence is
```

Comment:
I wonder if we should internally change the logic to "oneOf", i.e. erroring out on requests with multiple "repo sources" defined. This is just confusing.

Reply:
I had the same thought, but DNF does not enforce this AFAICT...

Reply:
Nevertheless I agree that it is confusing 😉